spyder/plugins/layout/plugin.py

```diff
@@ -130,6 +132,9 @@ def register(self):
         self._update_fullscreen_action()
 
     def before_mainwindow_visible(self):
+        # Update layout menu
+        self.get_container().update_layout_menu_actions()

```

Comment:
```suggestion
        self.update_layout_menu_actions()
```

Give that you added this method below, I think we should use it here instead.